debian/scripts/usr/local/bin/config_save

```diff
@@ -0,0 +1,35 @@
+# Save current configurations to partition containing conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
```

Comment:
as this is run from a booted system, it should not make nodes - udev will do that for us

Reply:
Removed